app/views/restrooms/index.html.haml

```diff
@@ -26,20 +26,20 @@
     .col-sm-12.headroom
       %ul{:class => "restrooms-list", :id => "list"}
         = render @restrooms
-        = paginate @restrooms
+        != pagy_nav @pagy
```

Comment:
Is there a comparison here we could make to keep the `=` instead of `!=`?

Reply:
Yes! It can be rewritten in this way to maintain the `=` that was there previously. This is functionally the same as using `!=`.

```suggestion
        = pagy_nav(@pagy).html_safe
```

Originally, the `!=` [evaluates the Ruby without ever sanitizing the HTML](https://haml.info/docs/yardoc/file.REFERENCE.html#unescaping_html). Rewritten, the `=` is maintained but it's [assumed to be safe](https://api.rubyonrails.org/v6.1.7.3/classes/String.html#method-i-html_safe) by marking it with `html_safe`. It has to be done in this format because pagy is an "[agnostic pagination gem it does not use the specific html_safe rails helper on its output](https://ddnexus.github.io/pagy/docs/how-to/#ignore-brakeman-unescapedoutputs-false-positives-warnings)".